src/WebJobs.Extensions.DurableTask/Bindings/BindingHelper.cs

```diff
@@ -37,6 +37,14 @@ public string DurableOrchestrationClientToString(IDurableOrchestrationClient cli
                 BaseUrl = this.config.HttpApiHandler.GetBaseUrl(),
                 RequiredQueryStringParameters = this.config.HttpApiHandler.GetUniversalQueryStrings(),
             };
+
+            if (this.config.Options.LocalRpcEndpointEnabled != false)
+            {
+                // If this field is missing, the out-of-proc SDK is expected to either fail
```

Comment:
I agree we should pick a standard for this, and we can document that standard. I added this comment because we already know that JS is going to do one thing (revert to old behavior) whereas Python will do something else (fail). The only reason to have JS revert to the old behavior is for backwards compatibility with older versions of the extension. New languages should always expect that the RPC endpoint will be available.